### DISCUSSION TITLE
Problems with CSV importer - maybe Layer 8 Problems - can somebody help?

### DISCUSSION THREAD

**Discussion by BlinderUhu** [firefly-iii/firefly-iii#5010] [Firefly III]
Hey,
i'm trying to use the importer to import my bank csv files. I already have read the "configure installation" page multiple times and also the "uploading files" and docker documentation but i don't now where the problem is. Can someboy please help me? :)

nice to know: 
- i'm using firefly III with docker on my raspberry pi
- also the importer runs on Docker  
-  i tryed to diverent files 

Thank you, and regards 

![image](https://user-images.githubusercontent.com/50677319/130331208-38551b0f-bbb8-421a-956d-6aa7a93333ef.png)

![image](https://user-images.githubusercontent.com/50677319/130330931-65700c84-d563-4209-a82c-15dca6cf863d.png)

**Comment by JC5** [comment]
Could you share one or two lines of your CSV file, and your configuration? Feel free to censor things, but names and IBANs only (those don't seem to be the problem).

**Reply by BlinderUhu** [reply]
thank you for the fast reply. If you need more Informations just let me know :)

CSV No.1:
"DE10000000000000000000";"20.08.21";"20.08.21";"KARTENZAHLUNG";"SVWZ+2021-08-19T17.25Debitk.2 2021-12ABWA+REWE SAGT DANKE. 44400190//Burghafen/DE";"REWE Markt GmbH";"DE10000000000000000000";"HYVEDEMMXXX";"-15,00";"EUR";"Umsatz gebucht"
"DE10000000000000000000";"19.08.21";"19.08.21";"KARTENZAHLUNG";"SVWZ+2021-08-18T12.44Debitk.2 2021-12ABWA+NETTO MARKEN-DISCOU//BADHABITS/DE";"Netto Marken-Discoun";"DE10000000000000000000";"HYVEDEMM447";"-0,91";"EUR";"Umsatz gebucht"

Config-File no. 1:

{
    "date": "d.m.Y",
    "default_account": 1,
    "delimiter": "semicolon",
    "headers": true,
    "rules": true,
    "skip_form": false,
    "add_import_tag": true,
    "specifics": [],
    "roles": [
        "account-iban",
        "date_book",
        "_ignore",
        "_ignore",
        "_ignore",
        "opposing-name",
        "opposing-iban",
        "_ignore",
        "amount",
        "currency-name",
        "_ignore"
    ],
    "do_mapping": {
        "9": true,
        "0": false,
        "1": false,
        "2": false,
        "3": false,
        "4": false,
        "5": false,
        "6": false,
        "7": false,
        "8": false,
        "10": false
    },
    "mapping": {
        "9": {
            "EUR": 1
        }
    },
    "duplicate_detection_method": "classic",
    "ignore_duplicate_lines": true,
    "ignore_duplicate_transactions": true,
    "unique_column_index": 0,
    "unique_column_type": "internal_reference",
    "version": 2
}

CSV No.2: 
![image](https://user-images.githubusercontent.com/50677319/130332097-ba214808-037a-4023-a239-add01caa1214.png)

Config-File no. 2:

{
    "date": "d.m.Y",
    "default_account": 2,
    "delimiter": "semicolon",
    "headers": false,
    "rules": true,
    "skip_form": false,
    "add_import_tag": true,
    "specifics": [],
    "roles": [
        "date_book",
        "_ignore",
        "description",
        "_ignore",
        "_ignore",
        "_ignore",
        "_ignore",
        "_ignore",
        "amount",
        "currency-name"
    ],
    "do_mapping": {
        "9": true,
        "0": false,
        "1": false,
        "2": false,
        "3": false,
        "4": false,
        "5": false,
        "6": false,
        "7": false,
        "8": false
    },
    "mapping": {
        "9": {
            "EUR": 1
        }
    },
    "duplicate_detection_method": "classic",
    "ignore_duplicate_lines": true,
    "ignore_duplicate_transactions": true,
    "unique_column_index": 0,
    "unique_column_type": "internal_reference",
    "version": 2
}

**Reply by JC5** [reply]
Try "y" in the date of the first file, not "Y".

**Reply by BlinderUhu** [reply]
Unfortunately this did not work :( . Now i get this error trying importing the first file

![image](https://user-images.githubusercontent.com/50677319/130332575-3542fa2d-6a09-4819-b6ea-ca5691cb3616.png)

**Reply by JC5** [reply]
I see it now. Use the transaction date ("date") column role, not the booking date.

**Reply by BlinderUhu** [reply]
It worked 🎊 Thank you so much 💯

**Reply by JC5** [reply]
Good to hear! No problem! 🥳
